src/main/java/com/github/amnotbot/BotCommandInterpreter.java

```diff
@@ -29,39 +29,32 @@
 import java.util.HashMap;
 import java.util.Iterator;
 import java.util.LinkedList;
-import java.util.regex.Matcher;
+import java.util.List;
 import java.util.regex.Pattern;
 
-import org.apache.commons.lang.StringUtils;
-
 import com.github.amnotbot.config.BotConfiguration;
 import com.github.amnotbot.spam.BotSpamDetector;
 
 public class BotCommandInterpreter
 {
-    private String cmdTrigger;
+    private String helpTrigger;
     private BotSpamDetector spamDetector;
-    
-    /** Collection of commands to fire for links. */
-    private LinkedList<BotCommand> linkListeners;
-    
-    /** Map of commands */
-    private HashMap<BotCommandEvent, LinkedList<BotCommand>> cmdListeners;
 
-    public BotCommandInterpreter(BotSpamDetector spamDetector)
+    private HashMap<BotCommandEvent, List<BotCommand>> cmdListeners;
+
+    public BotCommandInterpreter()
     {    
-        this.spamDetector = spamDetector;
-        this.cmdTrigger =
-                BotConfiguration.getConfig().getString("command_trigger", ".");
+        this.spamDetector = new BotSpamDetector();
+        this.helpTrigger =
```

Comment:
Well, I tried not to go beyond famous column number 80. If that is what you are referring to.